apps/hauptstadt/pages/a/[slug].tsx

```diff
@@ -55,7 +55,7 @@ export default function ArticleBySlugOrId() {
         </ArticleWrapper>
       )}
 
-      {!data?.article?.disableComments && (
+      {data?.article && !data.article.disableComments && (
```

Comment:
Changing this fixes runtime errors during SSG which allows to preview draft articles